From e0e9090106af6ae42c5a17a1c5d8edffb4d46ba1 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Thu, 13 Jan 2022 10:50:16 +0100
Subject: [PATCH 405/407] arm64: dts: rk3399-pinephone-pro: Fix modem-power
 support

Some things were copy pasted wrong from original Pinephone dts.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 313077d0f4ad..61c9907643e8 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -1069,7 +1069,7 @@ modem {
 		power-supply = <&vcc_4g>;
 		vbus-supply = <&vcc_4g_5v>;
 
-		enable-gpios = <&gpio0 RK_PB0 GPIO_ACTIVE_LOW>; // W_DISABLE#
+		enable-gpios = <&gpio0 RK_PB0 GPIO_ACTIVE_HIGH>; // W_DISABLE#
 		reset-gpios = <&gpio3 RK_PB0 GPIO_ACTIVE_HIGH>;
 		status-gpios = <&gpio3 RK_PA6 GPIO_ACTIVE_HIGH>;
 		pwrkey-gpios = <&gpio0 RK_PB5 GPIO_ACTIVE_HIGH>;
@@ -1081,7 +1081,7 @@ modem {
 		cts-gpios = <&gpio3 RK_PC0 GPIO_ACTIVE_HIGH>;
 		rts-gpios = <&gpio3 RK_PC1 GPIO_ACTIVE_HIGH>;
 
-		quectel,qdai = "1,1,0,1,0,0,1,1";
+		quectel,qdai = "3,0,0,4,0,0,1,1";
 	};
 };
 
-- 
2.31.1

